documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -9,80 +9,92 @@
+The funding of public goods on Namada will be conducted through a structure we call "Public Goods Stewards".
 
-Namada encourages the public goods council to adopt a default social consensus of an equal split between categories, meaning 1.25% per annum inflation for each category (e.g. 1.25% for technical research continuous funding, 1.25% for technical research retroactive PGF). If no qualified recipients are available, funds may be redirected or burnt.
+Each steward is elected by governance through separate governance proposals. Each steward will be responsible for covering a specific area of public goods, which they describe during their election. Stewards can then "propose" funding of various public goods, which will pass by default. However, Governance retains the power to veto any proposal, which would result in the Steward being removed from the set of Stewards.
 
-The Namada PGF council is also granted a 5% income as a reward for conducting PGF activities (5% * 10% = 0.05% of total inflation). This will be a governance parameter subject to change.
+## Voting for the Steward
 
-## Voting for the Council
+### What is a Steward (technically)?
+All valid PGF stewards will be established multisignature account addresses. These must be created by the intdended parties (which may very well be just one person, but could be more) that wish to represent the Steward entity. For example, if David Alice and Bob wish to represent the combined steward DAB, they may do so as a common entity. But likewise, Alice can create her own 1-out-of-1 multisig that to just represent herself.
 
-### Constructing the council
-All valid PGF councils will be established multisignature account addresses. These must be created by the intdended parties that wish to create a council. The council will therefore have the discretion to decide what threshold will be required for their multisig (i.e the "k" in the "k out of n").
 
+### Becoming a Steward
+The first step towards becoming a Steward is to instantiate a multisignature account. This is done through the CLI.
 
-### Proposing Candidacy
-The council will be responsible to publish this address to voters and express their desired `spending_cap`.
+In order to propose candidacy as a PGF Steward, the Steward must initiate a custom governance proposal. At the cost of deposited NAM, the governance proposal is broadcasted on-chain and governance is able to vote on whether the Steward-applicant will be accepted or not. Together with this proposal, the applicant is encouraged to provide a motivational statement as to why they should be entrusted with the responsibility of proposing public goods candidates. This will also include a commitment to at least one of the categories of public goods funding that social consensus has established (or propose their own category, which would inherintly introduce a new category into social consensus, should the proposal be accepted).
```

Comment:
```suggestion
In order to propose candidacy as a PGF Steward, the steward must initiate a custom governance proposal. At the cost of deposited NAM, the governance proposal is broadcast on-chain and governance is able to vote on whether the steward-applicant will be accepted or not. Together with this proposal, the applicant is encouraged to provide a motivational statement as to why they should be entrusted with the responsibility of proposing public goods candidates. This will also include a commitment to at least one of the categories of public goods funding that social consensus has established (or propose their own category, which would inherently introduce a new category into social consensus, should the proposal be accepted).
```